examples/connectivity/plot_mne_inverse_envelope_correlation.py

```diff
@@ -86,7 +86,8 @@
 # Compute the degree and plot it
 # ------------------------------
 
-degree = mne.connectivity.degree(corr, 0.15)
+threshold_percent = 0.15  # percentage of strongest edges to keep in the graph
```

Comment:
This is not a percentage but a proportion

Reply:
what do you suggest? put 15 and not 0.15?

Reply:
I prefer `0.15` so maybe `threshold_proportion` or `threshold_prop`? (although I also frequently refer to proportional values as percentages, so although it is not strictly correct it seems intuitive to me)